src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource.java

```diff
@@ -686,8 +688,26 @@ private void doRetrieve(SCMSourceCriteria criteria, SCMHeadObserver observer, Ta
                 }
                 branches++;
             }
-            listener.getLogger().format("%n  %d branches were processed%n", branches);
+
+            listener.getLogger().format("%n  %d branches were processed%n%n", branches);
+
+            List<GHTag> tagList = repo.listTags().asList();
+
+            for (GHTag tag: tagList) {
+                SCMHead taghead = new BranchSCMHead(tag.getName());
```

Comment:
@stephenc thanks! accidentally pushed this to the origin repo but still experimenting with how this would all work.